src/plugins/socketmon/socketmon.cpp

```diff
@@ -1092,25 +721,36 @@ socketmon::socketmon(drakvuf_t drakvuf, const socketmon_config* c, output_format
     json_object* tcpip_profile_json = json_object_from_file(c->tcpip_profile);
     if (!tcpip_profile_json)
     {
-        PRINT_DEBUG("Socketmon plugin fails to load JSON debug info for tcpip.sys\n");
+        PRINT_DEBUG("Socketmon plugin fails to load JSON profile for tcpip.sys\n");
```

Comment:
There seems to be a couple of these where you seem to be reverting changes made in recent merges.

Reply:
I'm very sorry. I will fix it and search for other places.

Let me know if you would find other such places please.

Reply:
I think I have cleaned all such spaces.